server-sent-events/Dockerfile

```diff
@@ -0,0 +1,86 @@
+# ================================
+# Build image
+# ================================
+FROM swift:5.10-jammy as build
```

Comment:
6.0?

Reply:
Needed as the new inbound close handler requires 6.0